packages/amplify-category-function/src/provider-utils/awscloudformation/utils/environmentVariablesHelper.ts

```diff
@@ -0,0 +1,116 @@
+import path from 'path';
+import _ from 'lodash';
+import { $TSContext, pathManager, JSONUtilities } from 'amplify-cli-core';
+import { categoryName, functionParametersFileName } from './constants';
+
+export const getStoredEnvironmentVariables = (context: $TSContext, resourceName: string): object => {
```

Comment:
this return type should be `Record<string, string>`. There are several other places where the `object` type is used that should be typed with this as well